views/block.tmpl

```diff
@@ -84,9 +84,9 @@
                 <div class="col-7 col-sm-8 text-left">
                     <span class="text-secondary fs13">Block Time</span>
                     <br>
-                    <span class="fs18 font-weight-bold lh1rem d-inline-block pt-1">{{.BlockTime.PrettyMDY}}</span>
+                    <span class="fs18 font-weight-bold lh1rem d-inline-block pt-1">{{.BlockTime.PrettyMDY}} <span class="fs13 text-secondary">{{.BlockTime.HMSTZ}}</span></span>
                     <br>
-                    <span class="fs14 text-secondary">{{.BlockTime.HMSTZ}}</span>
+                    <span class="text-secondary fs13"><span data-target="time.age" data-age="{{.BlockTime.UNIX}}"></span> ago</span>
```

Comment:
Please add `jsonly` to the outer/fs13 span class list.

Reply:
Done